lib/src/components/NxPageHeader/__tests__/NxPageHeader.test.tsx

```diff
@@ -23,13 +23,13 @@ describe('NxPageHeader', function() {
 
   it('renders the default logo within the header', function() {
     const view = quickRender(),
-        logo = view.getByRole('img');
+        logo = view.getAllByRole('img')[0];
```

Comment:
Both the light mode and dark mode logos were being read here, so changed to `getAllByole('img')[0]` to select just one